packages/financial-templates-lib/src/price-feed/InsuredBridgePriceFeed.ts

```diff
@@ -69,10 +69,13 @@ export class InsuredBridgePriceFeed extends PriceFeedInterface {
       depositHash: string;
     }
     let matchedRelay: MatchedRelay | undefined;
+    const latestL1Block = await this.l1Client.l1Web3.eth.getBlockNumber()
```

Comment:
This pricefeed is only used in `getHistorifcalPrice.js` script I believe but pls double check me